MyAddressBook-starter/app/src/main/java/com/example/android/myaddressbook/ContactsActivity.kt

```diff
@@ -0,0 +1,403 @@
+/*
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.android.myaddressbook
+
+import android.annotation.SuppressLint
+
+class ContactsActivity : AppCompatActivity(), TextWatcher {
+    private lateinit var mContacts: ArrayList<Contact>
+    private var mAdapter: ContactsAdapter? = null
+    private var mPrefs: SharedPreferences? = null
+    private var mFirstNameEdit: EditText? = null
+    private var mLastNameEdit: EditText? = null
+    private var mEmailEdit: EditText? = null
+    private var mEntryValid = false
+    @Override
+    protected fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_contacts)
+        val toolbar: Toolbar = findViewById(R.id.toolbar)
```

Comment:
Try to use https://medium.com/androiddevelopers/use-view-binding-to-replace-findviewbyid-c83942471fc instead of `findViewById `